### ISSUE TITLE
Configuration option for disabling aria-label

### ISSUE THREAD

**Issue by gumerov-amir** [HEIGE-PCloud/DoIt#1336]
### Describe the feature you want 描述你的功能需求

I use screenreader, and, when i read post's headings, i hear something this "Header mark for 'Music'  link    Music  heading    level 6". it'll be better to have option to disable aria-label in file layout/_default/_markup/render-heading.html in line     <a href="#{{ .Anchor | safeURL }}" class="header-mark" aria-label="Header mark for '{{ .Text }}'"></a>

or option for disabling toc and this anchor links

**Comment by HEIGE-PCloud** [comment]
Do you think it's safe to remove it completely? Or even mark it with [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden)?

**Comment by CXwudi** [comment]
I think [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) works for this case.

**Comment by gumerov-amir** [comment]
i do not think, that it is a good idea, because it's usefull to have an ability to copy a link, but this aria-label should be deleted
